mapie/classification.py

```diff
@@ -341,33 +534,122 @@ def predict(
         (n_samples,) and (n_samples, n_classes, n_alpha) if alpha is not None.
         """
         # Checks
+        include_last_label = self._check_include_last_label(include_last_label)
         alpha_ = check_alpha(alpha)
         check_is_fitted(
             self,
             [
                 "single_estimator_",
-                "scores_",
+                "conformity_scores_",
                 "n_features_in_",
-                "n_samples_val_"
-            ]
+                "n_samples_val_",
+            ],
         )
         X = check_array(X, force_all_finite=False, dtype=["float64", "object"])
         y_pred = self.single_estimator_.predict(X)
         y_pred_proba = self.single_estimator_.predict_proba(X)
+        y_pred_proba = self._check_proba_normalized(y_pred_proba)
         n = self.n_samples_val_
         if alpha_ is None:
             return np.array(y_pred)
         else:
             check_alpha_and_n_samples(alpha_, n)
             self.quantiles_ = np.stack([
                 np.quantile(
-                    self.scores_,
-                    ((n + 1) * (_alpha)) / n,
-                    interpolation="lower"
+                    self.conformity_scores_,
+                    ((n + 1) * (1 - _alpha)) / n,
+                    interpolation="higher"
                 ) for _alpha in alpha_
             ])
-            prediction_sets = np.stack([
-                y_pred_proba > (quantile)
-                for quantile in self.quantiles_
-            ], axis=2)
+            if self.method == "score":
+                prediction_sets = np.stack(
+                    [
+                        y_pred_proba > 1 - quantile
+                        for quantile in self.quantiles_
+                    ],
+                    axis=2,
+                )
+            elif self.method == "cumulated_score":
+                # sort labels by decreasing probability
+                index_sorted = np.fliplr(np.argsort(y_pred_proba, axis=1))
```

Comment:
Pour toute cette méthode, peut-on homogénéiser les notations avec la partie `fit` ?

Par exemple :

* `index_sorted` -> `index`
* `y_proba_sorted` -> `y_pred_proba_sorted`
* `y_proba_cumsum_sorted` -> `y_pred_proba_sorted_cumsum`
* `y_proba_sorted_last` -> `y_pred_proba_sorted_last`

Reply:
Bien vu Grégoire, les 3 méthodes sont maintenant homogénéisées.